index.html

```diff
@@ -701,6 +709,12 @@ <h3>Legal Identity</h3>
 comfortable vouching for them and resettling them in a safer area for the
 duration of the conflict.
         </dd>
+        <dt>
+<udef>L.5 Entitled visitor</udef>
+        </dt>
+        <dd>
+Some historic tourist sites that charge an entrance fee, are also used daily by local people who do not need to pay to enter. The site administration issues credentials to the local people who are entitled to enter the site free of charge. These credentials are marked "non-transferable" to ensure that the local people do not pass their credential onto someone else (who would normally have to pay the entrance fee).
+        </dd>
```

Comment:
```suggestion
        <p class="shortUcContributor">Contributed by David Chadwick</p>
        </dd>
```

Pick one of these two use cases, with the credit line, and I think we'll be good to accept.

FWIW, we like the shopper more than the tourist, as it feels more accessible.

Reply:
Ok. I will keep the CostCo example.